tests/sqllogictests/suites/temp_table/temporary_tables_table.sql

```diff
@@ -0,0 +1,22 @@
+statement ok
+CREATE TEMP TABLE t0(a int not null, b int not null);
```

Comment:
Why this test do not add `onlyif mysql` tag?

Reply:
https://github.com/datafuselabs/databend/blob/f885364b8c4eefe804534e007637b9378fe4ea36/scripts/ci/ci-run-sqllogic-tests.sh#L22-L25

Reply:
https://github.com/datafuselabs/databend/blob/f885364b8c4eefe804534e007637b9378fe4ea36/scripts/ci/ci-run-sqllogic-tests.sh#L23C1-L24C1